index.bs

```diff
@@ -697,53 +702,68 @@ When this method is invoked, the user agent MUST execute the following algorithm
         |credTypesAndPubKeyAlgs|, |excludeCredentialDescriptorList|, and |authenticatorExtensions| as parameters.
     1. [=set/Append=] |authenticator| to |issuedRequests|.
 
-1. Start a timer for |adjustedTimeout| milliseconds. Then execute the following steps [=in parallel=]. The [=task source=] for
-    these [=tasks=] is the [=dom manipulation task source=].
+1. Start a timer for |adjustedTimeout| milliseconds.
 
-1. While |issuedRequests| [=list/is not empty=], perform the following actions depending upon the |adjustedTimeout| timer and
-    responses from the authenticators:
+1. [=While=] |issuedRequests| [=list/is not empty=], perform the following actions in parallel depending upon the |adjustedTimeout| 
+    timer and responses from the authenticators but abort if the operation is [=terminated=]:
```

Comment:
When synchronising parallel actions, instead of using a loop, I've "waited" on values. I'm not sure if this is a better system or not, @annevk what do you think?

In https://wicg.github.io/background-fetch/#attempt-a-background-fetch-algorithm I do something similar, but I set up variables like "completeFetches" (step 2) and "abandoned" (step 3). Step 6 does a load of parallel tasks, and may set the "abandoned" flag. Step 7 waits for these values to change.